app/models/teacher_profile.rb

```diff
@@ -20,6 +20,8 @@ class TeacherProfile < ApplicationRecord
 
   self.filter_attributes += [:trn]
 
+  scope :order_by_created_at, -> { order(created_at: "asc") }
```

Comment:
I don't feel strongly about it, but it may be worth being explicit to the direction here; something like `oldest_first`?

Reply:
Done, thanks :)